xilem/src/window_view.rs

```diff
@@ -1,8 +1,8 @@
 // Copyright 2025 the Xilem Authors
 // SPDX-License-Identifier: Apache-2.0
 
+use masonry::app::RenderRoot;
```

Comment:
is this similar enough?
 https://github.com/linebender/xilem/blob/f97d19157a1930a9c08d5c7b05d968154b53a4fd/xilem/src/driver.rs#L13

Reply:
Similar enough how? Those things live in `masonry_winit`.